addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,52 @@
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col pb24 text-center" data-name="Heading">
+                    <h3>Latest news</h3>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative">
+                <div class="s_timeline_line position-absolute top-0 bottom-0 d-block border-start border-300"/>
+                <div class="s_timeline_row position-relative d-flex gap-5 align-items-start pb-5" data-name="Year">
+                    <span class="s_timeline_dot o_not_editable position-absolute translate-middle-x d-flex rounded-circle bg-o-color-1" contenteditable="false"/>
+                    <div class="s_timeline_content w-100 ps-4 ps-md-0" data-name="Date">
+                        <div class="s_timeline_card text-md-end">
+                            <small class="text-muted">13/06/2019</small>
+                            <h4>First Feature</h4>
+                            <p>A timeline is a graphical representation on which important events are marked.</p>
+                            <a href="#" class="btn btn-primary">See more</a>
                         </div>
                     </div>
-                    <i class="fa fa-1x fa-graduation-cap bg-o-color-2 rounded-circle s_timeline_icon"/>
+                    <div class="s_timeline_content w-100"/>
                 </div>
-                <div class="s_timeline_content d-flex">
-                    <i class="fa fa-1x fa-asterisk bg-o-color-2 rounded-circle s_timeline_icon"/>
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">...and switch the timeline contents to fit your needs.</p>
+                <div class="s_timeline_row position-relative d-flex gap-5 flex-md-row-reverse align-items-start" data-name="Year">
```

Comment:
`data-name="Year"` looks unnecessary. Be aware of empty editor boxes ;)
![image](https://github.com/user-attachments/assets/d66f8637-4f70-4264-9e7b-0399ab1c4e7a)

Reply:
It's even worst without 😬
![Capture d’écran 2024-07-26 à 07 37 36](https://github.com/user-attachments/assets/52ab95eb-4fe1-4f0e-b7ad-acd6d82b4063)

I'm not sure we can hide it in the WB since the user can select it to change the order.
![Capture d’écran 2024-07-26 à 07 39 57](https://github.com/user-attachments/assets/fca3b4ad-fc71-4167-87b1-60e8594b9b4e)

I would keep it for now and will adapt the name of the other block (Date).

Reply:
Not related to `data-name` indeed , we have empty sections in master as well. 
Let's check with the devs.
![image](https://github.com/user-attachments/assets/5d835e28-7014-4232-8a75-8793df3f4f15)